### PR TITLE
fix: Update faq.md with bcrypt generator website

### PR DESCRIPTION
**Pull request by cabrinha**
previous link was broken

Checklist:

* [c] Either (a) I've created an [enhancement proposal](https://github.com/argoproj/argo-cd/issues/new/choose) and discussed it with the community, (b) this is a bug fix, or (c) this does not need to be in the release notes.
* [ ] The title of the PR states what changed and the related issues number (used for the release note).
* [ ] I've updated both the CLI and UI to expose my feature, or I plan to submit a second PR with them.
* [ ] Optional. My organization is added to the README.
* [x] I've signed the CLA and my build is green ([troubleshooting builds](https://argoproj.github.io/argo-cd/developer-guide/ci/)).

### CODE REVIEW

**Comment by jannfis** [comment]
Hi @cabrinha - thanks for your PR. Can you please share a little more info why this change would be necessary, tho? 

The bcrypt generator currently linked from the docs seems to work fine, and the difference between the old one and the new one is also, that the new one does bcrypt calculation server side (using POST to send your data to their server), while the current one is apparently pure client side (uses javascript to hash your data).

File: docs/faq.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ As workaround Argo CD allows providing [health check](operator-manual/health.md)
 By default the password is set to the name of the server pod, as per [the getting started guide](getting_started.md).
 
 To change the password, edit the `argocd-secret` secret and update the `admin.password` field with a new bcrypt hash. You
-can use a site like https://www.browserling.com/tools/bcrypt to generate a new hash. For example:
+can use a site like https://bcrypt-generator.com/ to generate a new hash. For example:
```
Comment (alexmt) [inline docs/faq.md]: I've noticed that https://bcrypt-generator.com/ uses 12 rounds by default (instead of 10). Can you please mention 10 rounds in docs.

**Review by alexmt — CHANGES_REQUESTED** [review]

**Comment by cabrinha** [comment]
> Hi @cabrinha - thanks for your PR. Can you please share a little more info why this change would be necessary, tho?
> 
> The bcrypt generator currently linked from the docs seems to work fine, and the difference between the old one and the new one is also, that the new one does bcrypt calculation server side (using POST to send your data to their server), while the current one is apparently pure client side (uses javascript to hash your data).

The site: https://www.browserling.com/tools/bcrypt doesn't resolve for me. If there is another site that would be better to use, or if the site is working for you, let me know.

File: docs/faq.md (hunk 1 above)
Comment (cabrinha) [reply docs/faq.md]: Sorry, I don't understand why 12 rounds is better/worse than 10 rounds, does it need to be 10? Sorry for the question 😅

**Comment by jannfis** [comment]
Hi @cabrinha, for me the tool at browserling.com works well, this is why I asked in the first place. I just used `app.uptrends.com` to check availability from various locations in the world, this was the result:

image

Maybe it was just a hiccup on your side?

**Comment by cabrinha** [comment]
> Hi @cabrinha, for me the tool at browserling.com works well, this is why I asked in the first place. I just used `app.uptrends.com` to check availability from various locations in the world, this was the result:
> 
> image
> 
> Maybe it was just a hiccup on your side?

Thanks for clarifying, I'll close this PR 😅

**Pull request closed without merging**